.github/workflows/tests.yml

```diff
@@ -14,6 +14,10 @@ jobs:
   unit-tests:
     name: unit tests
     runs-on: ubuntu-20.04
+    #strategy:
```

Comment:
@deanlee in the future, you can uncomment this to run it several times

Reply:
ok!